docs/packages/client/overview.md

```diff
@@ -141,95 +122,73 @@ We provide a partially typed experience for GraphQL, because we do not want to l
 how you are going to query the remote system. Take a look at this example:
 
 ```typescript
-declare module 'fastify' {
-  interface GraphQLQueryOptions {
-    query: string;
-    headers: Record<string, string>;
-    variables: Record<string, unknown>;
-  }
-  interface GraphQLClient {
-    graphql<T>(GraphQLQuery): PromiseLike<T>;
-  }
-  interface FastifyInstance {
-    'client'
-    : GraphQLClient;
-
-  }
-
-  interface FastifyRequest {
-    'client'<T>(GraphQLQuery): PromiseLike<T>;
-  }
-}
+export interface Movie {
+  'id'?: string;
+
+  'title'?: string;
 
-declare namespace client {
-  export interface Clientoptions {
-    url: string
-  }
-  export interface Movie {
-    'id'?: string;
+  'realeasedDate'?: string;
 
-    'title'?: string;
+  'createdAt'?: string;
 
-    'realeasedDate'?: string;
+  'preferred'?: string;
 
-    'createdAt'?: string;
+  'quotes'?: Array<Quote>;
 
-    'preferred'?: string;
+}
+export interface Quote {
+  'id'?: string;
 
-    'quotes'?: Array<Quote>;
+  'quote'?: string;
 
-  }
-  export interface Quote {
-    'id'?: string;
+  'likes'?: number;
 
-    'quote'?: string;
+  'dislikes'?: number;
 
-    'likes'?: number;
+  'movie'?: Movie;
 
-    'dislikes'?: number;
+}
+export interface MoviesCount {
+  'total'?: number;
 
-    'movie'?: Movie;
+}
+export interface QuotesCount {
+  'total'?: number;
 
-  }
-  export interface MoviesCount {
-    'total'?: number;
+}
+export interface MovieDeleted {
+  'id'?: string;
 
-  }
-  export interface QuotesCount {
-    'total'?: number;
+}
+export interface QuoteDeleted {
+  'id'?: string;
 
-  }
-  export interface MovieDeleted {
-    'id'?: string;
+}
 
-  }
-  export interface QuoteDeleted {
-    'id'?: string;
+interface GraphQLQueryOptions {
+  query: string;
+  headers: Record<string, string>;
+  variables: Record<string, unknown>;
+}
 
-  }
-  export const client: Clientplugin;
-  export { client as default };
+interface GraphQLClient {
+  graphql<T>(options: GraphQLQueryOptions): Promise<T>;
 }
 
-declare function client(...params: Parameters<Clientplugin>): ReturnType<Clientplugin>;
-export = client;
+export function generateQuotesClient(opts: PlatformaticClientOptions): Promise<GraphQLClient>;
+export default generateQuotesClient;
 ```
 
 Given only you can know what GraphQL query you are producing, you are responsible for typing
 it accordingly.
 
-## Usage with Standalone Fastify
-
-If a Platformatic configuration file is not found, a complete Fastify plugin is generated to be 
-used in your Fastify application like this:
+## Use the Fastify plugin
 
 ```js
 const fastify = require('fastify')()
-const client = require('./your-client-name')
+const pltClient = require('@platformatic/client/lib/fastify-plugin')
```

Comment:
it should be in the root, without lib/

We should also add some guidance about the types needed - my understanding is that we are not generating them anymore.

Reply:
Moved and guide added!